gradlew

```diff
@@ -1,164 +1,164 @@
-#!/usr/bin/env bash

```

Comment:
What actually changed in this file? Looks like you formatted the whole file differently which makes it impossible to determine the diff. In the future, please either avoid changing the formatting of files or do it in separate commits that state that its only formatting changes.